src/main/scala/za/co/absa/abris/avro/parsing/utils/AvroSchemaUtils.scala

```diff
@@ -104,3 +109,21 @@ object AvroSchemaUtils {
   }
 
 }
+
+case class DefaultAvroSchemaConverter() extends AvroSchemaConverter {
+  def toAvroType( catalystType: DataType,
+                  nullable: Boolean = false,
+                  recordName: String = "topLevelRecord",
+                  nameSpace: String = "")
+  : Schema= {
+    SchemaConverters.toAvroType(catalystType, nullable, recordName, nameSpace)
+  }
+}
```

Comment:
- case class is not really need it here, or is it? (Case classes may cause backward compatibility issues. Better to use them only when there is lot to gain from it)
- I would move this to its own file
- formatting